sdk/react/src/DVCProvider.tsx

```diff
@@ -11,17 +10,13 @@ type Props = {
 
 export default function DVCProvider(props: Props): React.ReactElement {
     const { envKey, user, options } = props.config
-    const [client, setClient] = useState<DVCClient | undefined>(undefined)
+    const client = useMemo(() => initializeDVCClient(envKey, user, options), [envKey, user, options])
```

Comment:
do we actually want to create a new client when these props change? Or should we make it so only one client can be created per instance of this Provider?

Reply:
Hmm, I’m not sure. Would those props be updated very often? It looks like the dependency array has to contain all the values referenced in the function. Or is there another way to do this so that a new client isn’t created every time those props are changed?